runtime/common/src/main/java/edu/snu/nemo/runtime/common/RuntimeIdGenerator.java

```diff
@@ -76,36 +79,25 @@ public static String generateStageId(final Integer stageId) {
   }
 
   /**
-   * Generates the ID for {@link edu.snu.nemo.runtime.common.plan.physical.Task}.
-   *
-   * @param irVertexId the ID of the IR vertex.
-   * @return the generated ID
-   */
-  public static String generateLogicalTaskId(final String irVertexId) {
-    return "Task-" + irVertexId;
-  }
-
-  /**
-   * Generates the ID for {@link edu.snu.nemo.runtime.common.plan.physical.Task}.
+   * Generates the ID for the physical representation of an IRVertex.
    *
-   * @param index         the index of the physical task.
-   * @param logicalTaskId the logical ID of the task.
+   * @param irVertexId the id of the IRVertex.
+   * @param index      the index of the physical task.
    * @return the generated ID
    */
-  public static String generatePhysicalTaskId(final int index,
-                                              final String logicalTaskId) {
-    return logicalTaskId + PHYSICAL_TASK_ID_SPLITTER + index;
+  public static String generatePhysicalIRVertexId(final String irVertexId,
```

Comment:
As far as I understand, this PR gets rid of the Tasks in runtime, and replaces it with the original IR vertices, which previous tasks have been referring to. However, this method creates ID for the Physical IR vertex which I'm unsure of its purpose. I think there should be rather a _Physical Vertex_ or something else and make it different from the original IR vertex, if it is not going to be used as it is used in the IR. Please check this part throughout the PR.

Reply:
Thanks! I've missed those. I've removed the related methods and variables, as they don't have a functional purpose.